regen-network/regen-web
# feat: APP-150 add featured projects sorting on /projects page

## Description

https://regennetwork.atlassian.net/browse/APP-150?atlOrigin=eyJpIjoiMjAwOGNjZThhMjc3NGQ1Zjk3ZTg2NmJiMTNmY2ViMWIiLCJwIjoiaiJ9



---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...


### How to test



From https://deploy-preview-2387--regen-marketplace.netlify.app/projects/1, check that the featured projects set on sanity (https://regen.sanity.studio/staging/structure/registry;home) appear first, then the projects from price low to high.
Since "community projects" filter were off by default, if any of the featured projects is a community project, it won't appear first when featured projects sorting is enabled. After some discussions with @clevinson we've decided that we could simply turn on this community projects filter by default for now (since there's no community projects on prod yet), have it set as an environment variable that we could easily turn off again if we want in the future.

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...
